examples/js/loaders/TDSLoader.js

```diff
@@ -43,9 +43,13 @@ THREE.TDSLoader.prototype = {
 
 		var scope = this;
 
-		var loader = new THREE.FileLoader( this.manager );
+		var path = scope.path === undefined ? THREE.Loader.prototype.extractUrlBase( url ) : scope.path;
+		
+		var loader = new THREE.FileLoader( scope.manager );
 
 		loader.setResponseType( 'arraybuffer' );
+		
+		loader.setPath( path );

```

Comment:
I think we don't need this line.

Reply:
That line sets the new path (deferred from URL) when there is no path defined

Reply:
Um, i've missed the different approach in this loader.

Reply:
Please consider this comment: https://github.com/mrdoob/three.js/pull/12614#discussion_r150025998

Wouldn't be the mentioned case also problematic for `TDSLoader`? 

\cc @donmccurdy